src/v2/Components/RecentlyViewed.tsx

```diff
@@ -47,47 +49,50 @@ export class RecentlyViewed extends React.Component<RecentlyViewedProps> {
 
                 <Spacer mb={3} />
 
-                <Carousel
-                  data={me.recentlyViewedArtworksConnection.edges}
-                  render={artwork => {
-                    const aspect_ratio = get(
-                      artwork,
-                      w => w.node.image.aspect_ratio,
-                      1
-                    )
+                <Box mx={isMobile ? "-20px" : 0}>
```

Comment:
This applies to all cases-- but I'm curious why the choice to use `isMobile` here instead of setting the margin based on the breakpoints?

I'm thinking of syntax [like this](https://github.com/artsy/force/blob/aa20dda94a03cb7027b96d097056424583a8e16f/src/v2/Apps/FeatureAKG/Components/FeaturedArtists.tsx#L24)!

Reply:
I can make that change where we aren't relying on another factor to determine the margins, e.g. the index of the carousel's image:

```
<Box ml={isMobile && index === 0 ? 2 : 0}>
          <ArtistCollectionEntity
                lazyLoad={index > 5}
                collection={slide}
           />
</Box>
```